tests/uiautomation/XAMLCatalogTest/TextFieldsTest.cs

```diff
@@ -40,6 +40,7 @@ public static void TearDown()
             TestCommon.QuitApp();
         }
 
+        [Ignore]
```

Comment:
This test is actually an expected exception test but I need more info from the framework on how to extract the inner exception instead of the framework exception.

---
In reply to: [84972796](https://github.com/Microsoft/WinObjC/pull/1204#discussion_r84972796) [](ancestors = 84972796)